views/layout.erb

```diff
@@ -10,82 +10,75 @@
   <meta name="description" content="Custom BLAST server provided by SequenceServer (https://sequenceserver.com)"/>
 
   <%= Rack::Csrf.metatag(env) %>
-  <!-- CSS -->
```

Comment:
I wonder if we still need separate layouts for results and search. search_layout.erb was introduced when ONLY search page was converted to tailwind, perhaps they can be merged together again?

Reply:
Hi @tadast, Yes. We can merged them into one, but we have a different top nav style between search page and report page. do we want to keep both styles or choose one?

**search page**
![image](https://github.com/user-attachments/assets/32ea4761-c9e1-4a68-96a2-3e7fbae6457f)

**report page**
![image](https://github.com/user-attachments/assets/0f61cbf8-a27d-4f23-b7d2-51731cd740db)

Reply:
updated